_includes/pages/sponsors/top_sponsors.html

```diff
@@ -1,7 +1,17 @@
 <ul class="cards-list">
-  {% for sp in site.data.sponsors limit:3 %}
+  <li class="sponsor-card">
+    <a href="https://manas.tech/" class="name">Manas.Tech</a>
+    <img src="/assets/manas-orange.svg" alt="Logo of Manas.Tech">
+    <span class="last-payment" title="Last payment">thousands of hours since 2012</span>
+    <span class="all-time" title="All-time total">Lots of ♥️</span>
+  </li>
```

Comment:
Oh sorry, I mean that Manas should be added to the data and not to the page

Reply:
As long as it's not in the data, it makes sense to add it as an extra. Maybe that's even better overall 🤷 
The manas contribution isn't as easy to quantify...